content/applications/sales/sales/send_quotations/deadline.rst

```diff
@@ -1,43 +1,54 @@
-===================================================
-Use quotations deadline to stimulate your customers
-===================================================
+==========================================
+Quotation deadlines to stimulate customers
+==========================================
 
-When sending quotations, it is important to set a deadline to encourage your customers to act.
-Indeed, this will stimulate them because they will be afraid of missing a good deal and it will also
-allow you to protect yourself in case you have to fulfill an order at a price that is no longer
-profitable for you.
+When sending quotations, it's important to set a deadline to encourage customers to act and confirm
+orders in a timely manner.
 
-Set a deadline
-==============
+Setting deadlines on quotations entices customers to finalize the process because they may fear
+that they'll miss out on a good deal. Quotation deadlines are also useful because they can be used
+as protection for a company, in case an order has to be fulfilled at a price that is no longer
```

Comment:
```suggestion
as protection for a company, in case an order was quoted at a price that is no longer
```

Reply:
I think I may leave this in the *present/future* tense, as it's my understanding that we should avoid *past* tense whenever we can, but I totally understand the suggestion